Vagrantfile

```diff
@@ -875,3 +875,40 @@ Vagrant.configure("2") do |config|
     trigger.on_error = :continue
   end
 end
+
+class VVVScripts < Vagrant.plugin(2, :command)
+  def self.synopsis
+    "Execute script from /config/homebin/"
+  end
+
+  def execute
+    with_target_vms(nil, single_target: true) do |vm|
+      @env.ui.output "#{$yellow}Executing #{$red}#{ARGV[1]}#{$creset}\n"
+      vm.action(:ssh_run, ssh_run_command: "/srv/config/homebin/#{ARGV[1]}" )
+    end
+  end
+end
+
+class VVVCommand < Vagrant.plugin(2, :command)
+  def self.synopsis
+    "Execute custom command inside site folder"
+  end
+
+  def execute
+    with_target_vms(nil, single_target: true) do |vm|
+      @env.ui.output "#{$yellow}Executing in #{$red}#{ARGV[1]}#{$yellow}: #{$red}#{ARGV[2]}#{$creset}\n"
```

Comment:
What does th `@` do?

Reply:
Without `@` doesn't work because we need to access this variable inside the class as I can remember. I am not Ruby's expert but is used also in other part of the vagrantfile.